tests/unit/src/test/scala/tests/DocumentSymbolSlowSuite.scala

```diff
@@ -0,0 +1,92 @@
+package tests
+
+object DocumentSymbolSlowSuite extends BaseSlowSuite("documentSymbol") {
+
+  testAsync("documentSymbol") {
+    for {
+      // start with code that does not parse (notice the first char in Main.scala)
+      _ <- server.initialize(
+        """|
+           |/metals.json
+           |{
+           |  "a": { }
+           |}
+           |/a/src/main/scala/a/Main.scala
+           |}package a
+           |import java.util.concurrent.Future // unused
```

Comment:
Let's remove unnecessary code like imports, only keep relevant declarations.

Reply:
```scala
object Outer {
  class Inner
}
```

Reply:
the less code to copy-paste for expect tests the better

Reply:
👍 sounds good, I've based this one off `DefinitionSuite` but I agree it can be way shorter